File: AtCoder/abc201/B.kt
fun main() {
    val n = readLine()!!.toInt()
    val arr = mutableListOf<Pair<Int, String>>()
    for (i in 0 until n) {
        val (name, height) = readLine()!!.split(" ")
        arr.add(Pair(height.toInt(), name))
    }
    arr.sortBy { it.first }
    print(arr[n - 2].second)
}

File: AtCoder/abc201/B.swift
struct Mountain {
	let name: String
	let height: Int
}

var arr = [Mountain]()

let s = readLine() ?? ""
let n = s.toInteger()
for _ in 1...n {
	let curr = readLine()?.split(separator: " ") ?? []
	arr.append(Mountain(name: String(curr[0]), height: String(curr[1]).toInteger()))
}
arr.sort() {
	$0.height > $1.height
}
print(arr[1].name)


extension String {
	func toInteger() -> Int {
		return Int(self) ?? 0
	}
}
